standalone/pruntime/app/src/main.rs

```diff
@@ -854,7 +893,30 @@ fn main() {
         panic!("Initialize Failed");
     }
 
-    rocket().launch();
+    let mut core_num: u8 = 6;
+    let args: Vec<_> = env::args().collect();
+    if args.len() > 1 {
+        let arg1 = &args[1];
+        core_num = arg1.parse::<u8>().unwrap();;
```

Comment:
How about to use an arg parser library, such as clap, structopt, for better maintainability and help messages.